### PR TITLE
LabWork4_VladislavMilchov

### PR DESCRIPTION
**Pull request by VladislavMilchov**

### CODE REVIEW

File: LabWork4_VladislavMilchov/Script.sql [hunk 1]
```diff
@@ -0,0 +1,35 @@
+select count(name) 
+from country
+where IndepYear >=1970
+
+select count(distinct continent)
+from country;
+
+
```
Comment (sslavova) [inline LabWork4_VladislavMilchov/Script.sql:8]: Below query is wrong - it will produce NULL.

File: LabWork4_VladislavMilchov/Script.sql [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+select sum(surfacearea)
+from country
+where Region;
+
```
Comment (sslavova) [inline LabWork4_VladislavMilchov/Script.sql:12]: Add rounding in below query.
Also add aliases to all aggregation functions.

File: LabWork4_VladislavMilchov/Script.sql [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+group by Continent 
+order by sum(SurfaceArea) desc limit 3;
+
+select 
+CountryCode  
+, avg(population)
+from city
+group by CountryCode
+order by avg(Population) asc;
+
```
Comment (sslavova) [inline LabWork4_VladislavMilchov/Script.sql:26]: It's a good practice to add upper to the columns when you don't know if the values inside consist of lower or upper cases. MySQL accepts both 'NLD' or 'nld', but most of the databases are case sensitive so there it's better to do it this way:
UPPER(CountryCode) = 'NLD'

**Review by sslavova — CHANGES_REQUESTED** [review]

File: LabWork4_VladislavMilchov/Script.sql (hunk 1 above)
Comment (VladislavMilchov) [reply LabWork4_VladislavMilchov/Script.sql]: Fixed

**Review by sslavova — APPROVED** [review]
Good job! :)

**Pull request merged**